src/port1.0/porttrace.tcl

```diff
@@ -213,18 +213,29 @@ namespace eval porttrace {
         }
 
         # Allow access to some Xcode specifics
-        allow trace_sandbox "/var/db/xcode_select_link"
-        allow trace_sandbox "/var/db/mds"
-        allow trace_sandbox [file normalize ~${macportsuser}/Library/Preferences/com.apple.dt.Xcode.plist]
-        allow trace_sandbox "$env(HOME)/Library/Preferences/com.apple.dt.Xcode.plist"
+        set xcode_paths {}
+        lappend xcode_paths "/var/db/xcode_select_link"
+        lappend xcode_paths "/var/db/mds"
```

Comment:
After some searching.. I think `/var/db/mds` is for Spotlight.
https://apple.stackexchange.com/questions/238776/what-is-the-purpose-of-mds-folder-on-a-mac
Not sure if we need to deny access to it. But since it's here in the first place (I assume to search for Xcode), I'll leave it as such.